admin-dev/themes/new-theme/scss/theme.scss

```diff
@@ -182,3 +182,78 @@ table.table {
     }
   }
 }
+
+//** Background color of the whole progress component
+$progress-bg:                 #f5f5f5 !default;
+//** Progress bar text color
+$progress-bar-color:          #fff !default;
+
+//** Default progress bar color
+$progress-bar-bg:             $brand-primary !default;
+//** Success progress bar color
+$progress-bar-success-bg:     $brand-success !default;
+//** Warning progress bar color
+$progress-bar-warning-bg:     $brand-warning !default;
+//** Danger progress bar color
+$progress-bar-danger-bg:      $brand-danger !default;
+//** Info progress bar color
+$progress-bar-info-bg:        $brand-info !default;
+
+$font-size-base: 12px;
+$line-height-base: 1.428571429;
+$line-height-computed: floor($font-size-base * $line-height-base);
+
+//
+// Progress bars
+// --------------------------------------------------
+
+// Outer container
+.progress {
```

Comment:
This should probably be wrapped in a parent class `.progress-modal` since these class names are quite generic it encloses the rules only to the modal content this way